src/Artsy/Router/Utils/RenderStatus.tsx

```diff
@@ -39,6 +33,32 @@ export const RenderReady: React.FC<{
   )
 }
 
+export const RenderError: React.FC<{
+  error: { status?: number; data?: any }
+}> = props => {
+  logger.error(props.error.data)
```

Comment:
Remind me, does this go to Sentry or paper-trail?

Reply:
Sentry. Papertrail mirror force's server logs